src/OpenTracing/IScope.cs

```diff
@@ -0,0 +1,95 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="AbstractTag.cs">
+//   Copyright 2017-2018 The OpenTracing Authors
+//   
+//   Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+//   in compliance with the License. You may obtain a copy of the License at
+//   
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+//   Unless required by applicable law or agreed to in writing, software distributed under the License
+//   is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+//   or implied. See the License for the specific language governing permissions and limitations under
+//   the License.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OpenTracing
+{
+    using System;
+
+    /// <summary>
+    ///     A <see cref="IScope" /> formalizes the activation and deactivation of a <see cref="ISpan" />, usually from a CPU
+    ///     standpoint.
+    ///     <para>
+    ///         Many times a <see cref="ISpan" /> will be extant (in that <see cref="ISpan.Finish()" /> has not been called)
+    ///         despite being in a
+    ///         non-runnable state from a CPU/scheduler standpoint. For instance, a <see cref="ISpan" /> representing the child
+    ///         side of an
+    ///         RPC will be unfinished but blocked on IO while the RPC is still outstanding. A <see cref="IScope" /> defines
+    ///         when a given
+    ///         <see cref="ISpan" /> <em>is</em> scheduled and on the path.
+    ///     </para>
+    ///     <para>
+    ///         Calling <see cref="IScope.Dispose" /> marks the end of the active period for the current thread and
```

Comment:
This might not build when XML docs are created (with RELEASE) as Dispose is on IDisposable. Not sure though.

Reply:
I believe it should since it's also on `IScope` (via `IDisposable` and lacking explicit implementation (can you do explicit implementation on an interface even?)). How can I verify, build release and check the *.xml files?

Reply:
you only have to call ".\build.ps1" in the project root. this will build with RELEASE and throw an error if the xml docs cannot be created properly.

Reply:
resolved